datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/cmsync/event/SdxCmSyncStartEvent.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.datalake.flow.datalake.cmsync.event;
+
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_START_EVENT;
+
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class SdxCmSyncStartEvent extends SdxEvent {
+    public SdxCmSyncStartEvent(Long sdxId, String sdxName, String userId) {
+        super(SDX_CM_SYNC_START_EVENT.event(), sdxId, sdxName, userId);
```

Comment:
why? I mean `SdxEvent` default selector would be the same

Reply:
I did it for getSdxName to work, as no other ctor does set the sdx name. I did not want to refactor here, neither wanted to introduce another ctor.
I used sdxName here. But maybe it is really not needed as the sole id we use in kibana is the datalake crn.
https://github.com/hortonworks/cloudbreak/blob/4482d8cecde8808fa1a418932585e6fb32e98bc8/datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/cmsync/SdxCmSyncActions.java#L47